src/app/containers/CpsRecommendations/RecommendationsPromo/index.jsx

```diff
@@ -1,25 +1,98 @@
-import React from 'react';
-import { C_GHOST } from '@bbc/psammead-styles/colours';
-import { GEL_SPACING_HLF } from '@bbc/gel-foundations/spacings';
-import { GEL_GROUP_3_SCREEN_WIDTH_MAX } from '@bbc/gel-foundations/breakpoints';
+import React, { useContext } from 'react';
+import pathOr from 'ramda/src/pathOr';
 import styled from '@emotion/styled';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import { getSerifMedium } from '@bbc/psammead-styles/font-styles';
+import { getPica } from '@bbc/gel-foundations/typography';
+import { C_EBON, C_METAL, C_GREY_2 } from '@bbc/psammead-styles/colours';
 import { shape, string, oneOfType } from 'prop-types';
 import Grid from '../../../components/Grid';
-import StoryPromo from '../../StoryPromo';
+import { ServiceContext } from '#contexts/ServiceContext';
+import RecommendationsImage from '../RecommendationsPromoImage';
 import { storyItem } from '#models/propTypes/storyItem';
+import useCombinedClickTrackerHandler from '../../StoryPromo/useCombinedClickTrackerHandler';
 
-const StyledStoryPromoWrapper = styled.div`
-  > div {
-    display: grid;
-    margin: ${GEL_SPACING_HLF} 0;
-    background-color: ${C_GHOST};
-    @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
-      margin: ${GEL_SPACING_HLF} 0;
-    }
+const StyledPromoWrapper = styled.div`
+  padding: ${GEL_SPACING};
+  margin-top: ${GEL_SPACING};
+  background-color: ${C_GREY_2};
+`;
+
+const ImageWrapper = styled.div`
+  display: inline-block;
+  position: relative;
+  width: 4.7rem;
+  vertical-align: top;
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    width: 6.8rem;
+  }
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    width: 7.5rem;
+  }
+`;
+
+const TextWrapper = styled.div`
+  display: inline-block;
+  width: calc(100% - 7.5rem);
+  padding: 0 ${GEL_SPACING};
+  vertical-align: top;
+  height: 100%;
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    padding: 0 ${GEL_SPACING_DBL};
+  }
+`;
+
+const Link = styled.a`
+  position: static;
+  color: ${C_EBON};
+  text-decoration: none;
+  overflow-wrap: break-word;
+
+  &:before {
+    bottom: 0;
+    content: '';
+    left: 0;
+    overflow: hidden;
+    right: 0;
+    top: 0;
+    white-space: nowrap;
+    z-index: 1;
+  }
```

Comment:
It looks like this `::before` functionality for the link isn't behaving the same as the previous recommendations implementation. I think this is meant cover the `a` tag over the top of the recommendation item, unless this was changed in the UX designs?

Before:

https://user-images.githubusercontent.com/4706195/133770709-9295ebac-33ae-488e-b310-455c93b0eaa7.mov


After:

https://user-images.githubusercontent.com/4706195/133770727-502d3d8e-bdc9-4e12-8d6f-0e32a6ce2af9.mov

Reply:
No, you're right. The user should be able to click on the image/text and be redirected.

Reply:
Good spot. Fixed in commit d8229fffe08f724c1059fbbfc6fe0350831c7eb9